text/tests/unittests/automm/configs/model/timm_image.yaml

```diff
@@ -0,0 +1,23 @@
+model:
+  names: "timm_image"
+  timm_image:
+    checkpoint_name: "swin_small_patch4_window7_224"
+#    checkpoint_name: "beit_base_patch16_224"
+#    checkpoint_name: "vit_small_patch16_384"
+#    checkpoint_name: "convmixer_1536_20"
+#    checkpoint_name: "efficientnet_b1"
+#    checkpoint_name: "resnet18"
+#    checkpoint_name: "legacy_seresnet18"
+#    checkpoint_name: "resnest14d"
+    mix_choice: "all_logits"
+    data_types:
+      - "image"
+    train_transform_types:
+      - "resize_shorter_side"
+      - "center_crop"
+    val_transform_types:
+      - "resize_shorter_side"
+      - "center_crop"
+    image_norm: "imagenet"
+    image_size: 224
+    max_img_num_per_col: 1
```

Comment:
remove comments + add new line

Reply:
Already fixed.

Reply:
Oh, I noticed that these files are in the `unittests` folder.

Reply:
Thanks. Fixed this time.